java/ql/src/change-notes/2024-03-04-sensitive-log-remove-null-from-sources.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* To reduce the number of false positives in the query "Insertion of sensitive information into log files" (java/sensitive-log), variables with names that contain "null" (case-insensitively) will no longer considered sources of sensitive information.
```

Comment:
```suggestion
* To reduce the number of false positives in the query "Insertion of sensitive information into log files" (`java/sensitive-log`), variables with names that contain "null" (case-insensitively) will no longer be considered sources of sensitive information.
```

Reply:
Building on top of @intrigus-lgtm's suggestion:

```suggestion
* To reduce the number of false positives in the query "Insertion of sensitive information into log files" (`java/sensitive-log`), variables with names that contain "null" (case-insensitively) are no longer considered sources of sensitive information.
```